test/models/playlist_test.rb

```diff
@@ -2,12 +2,17 @@
 
 class PlaylistTest < ActiveSupport::TestCase
 
+  setup do
+    @user = users(:andrea)
+  end
+
   def valid_playlist_params
     {
       name: 'name1',
       energy: 35,
       danceability: 56,
-      genre: "rock"
+      genre: "rock",
+      user_id: @user_id
```

Comment:
```suggestion
      user: @user
```

à noter que `user_id: @user.id` est valide, mais `@user_id` n'a pas l'air d'exister dans ton test